### PR TITLE
Web Survey on 20211231

### LINKED ISSUE #3729: seamonkey: security update to 2.53.12
### CVE IDs

CVE-2019-9811, CVE-2019-11709, CVE-2019-11711, CVE-2019-11712, CVE-2019-11713, CVE-2019-11715, CVE-2019-11717, CVE-2019-11719, CVE-2019-11729, CVE-2019-11730

### Other security advisory IDs

- Firefox ESR 60.8.0, [mfsa2019-22](https://www.mozilla.org/en-US/security/advisories/mfsa2019-22/)
- Thunderbird 60.8.0, [mfsa2019-23](https://www.mozilla.org/en-US/security/advisories/mfsa2019-23/)

### Description

See...

- Firefox ESR 60.8.0, [mfsa2019-22](https://www.mozilla.org/en-US/security/advisories/mfsa2019-22/)
- Thunderbird 60.8.0, [mfsa2019-23](https://www.mozilla.org/en-US/security/advisories/mfsa2019-23/)

### Patches

N/A

### PoC(s)

N/A

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

Survey for updates in the `base-web` and `extra-web` categories.

Package(s) Affected
-------------------

- `curl+32` v7.80.0
- `hugo` v0.91.2
- `nginx` v1.21.5
- `qownnotes` v21.12.8
- `seamonkey` v2.53.10.2
- `telegram-desktop` v3.4.0

Security Update?
----------------

Yes

Build Order
-----------

```
extra-optenv32/curl+32
extra-web/hugo
extra-web/nginx
extra-web/qownnotes
extra-web/seamonkey
extra-web/telegram-desktop
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
- [x] 32-bit Optional Environment `optenv32`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`
- [ ] 32-bit Optional Environment `optenv32`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-web/telegram-desktop/spec [hunk 1]
```diff
@@ -1,9 +1,9 @@
-VER=3.3.0
+VER=3.4.0
 # Update tg_owt to the latest Git snapshot when updating Telegram Desktop
 OWTVER=429a6869e4a164e0aad2d8657db341d56f9a6a6f
 SRCS="tbl::https://github.com/telegramdesktop/tdesktop/releases/download/v$VER/tdesktop-$VER-full.tar.gz \
       git::rename=tg_owt;commit=${OWTVER}::https://github.com/desktop-app/tg_owt"
-CHKSUMS="sha256::72c51d0f24c78f9362c334091fb2de71ecfe786e8d9076f9f4fca4925809489b \
+CHKSUMS="sha256::8bd9067ba0a044789aa28b321715ba60971dc27ae8d027ef1f9879093c331157 \
          SKIP"
 SUBDIR=tdesktop-$VER-full
```
Comment (WhiredPlanck) [inline extra-web/telegram-desktop/spec]: Shall we quote the path here ...?

**Review by WhiredPlanck — CHANGES_REQUESTED** [review]

File: extra-web/telegram-desktop/spec (hunk 1 above)
Comment (MingcongBai) [reply extra-web/telegram-desktop/spec]: Yes.

Comment (MingcongBai) [reply extra-web/telegram-desktop/spec]: Addressed. Resolving.

**Comment by WhiredPlanck** [comment]
Telegram LGTM on AMD64

**Comment by MingcongBai** [comment]
image

QOwnNotes requires intervention, we should opt-out by default.

**Comment by MingcongBai** [comment]
Other packages LGTM.

**Comment by WhiredPlanck** [comment]
QOwnNotes LGTM on AMD64
App metrics and heartbeat are disabled by default.
All architecture test builds done.

图片
图片
图片

**Review by cthbleachbit — APPROVED** [review]

**Review by WhiredPlanck — APPROVED** [review]

**Pull request merged**